beta/examples/tensorflow/classification/main.py

```diff
@@ -210,6 +241,21 @@ def run(config):
         'validation_freq': 1,
     }
 
+    if 'train' in config.mode and is_accuracy_aware_training_mode:
+
+        train_epoch_fn, validate_fn = get_training_validation_functions(train_dataset,
+            validation_dataset, callbacks, initial_epoch, validation_steps, train_steps)
+
+        # instantiate and run accuracy-aware training loop
+        acc_aware_training_loop = AdaptiveCompressionTrainingLoop(config.nncf_config, compression_ctrl,
+                                                            runner_cls=AccuracyAwareTrainingRunner)
```

Comment:
maybe it's too much responsibility for the user to care about proper combination of 
`AdaptiveCompressionTrainingLoop` and `AccuracyAwareTrainingRunner`.
This runner should have certain methods, there should be some interface, but currently we have no restrictions.
Or we could encapsulate list of possible runners in Enum.

Reply:
This is a workaround since we do not yet have a properly working `nncf.__backend__` attribute. The user won't have to specify Runner objects at all, it'll be done internally based on the runtime backend.